geode-for-redis/src/distributedTest/java/org/apache/geode/redis/internal/commands/executor/list/LPushDUnitTest.java

```diff
@@ -64,71 +71,54 @@ public static void tearDown() {
   }
 
   @Test
-  public void shouldDistributeDataAmongCluster() {
+  public void lpush_ShouldPushMultipleElementsAtomically()
```

Comment:
This test is not testing distributed behaviour, as all the operations are being done to a list residing in a bucket that does not move during the test, and no validation of the contents of the secondary copy is done. This would be a decent integration test to add to `AbstractLPushIntegrationTest` (possibly refactored a little to use `ConcurrentLoopingThreads` rather than the `ExecutorServiceRule`, just to be consistent with other Integration tests), but in order to test distributed behaviour, we need to be either moving buckets directly or crashing servers to cause them to move, and doing some verification that the contents of the secondary bucket after failing over matches what we expect.

Reply:
Yeah, this is a draft, I didn't claim it was done yet. But since I have tests passing locally that are failing in CI, I wanted to make sure this was also running in CI.

Reply:
Sorry for the premature review. I'll hold off on more comments until you give the thumbs up.